libs/sdk-core/src/lib/db/kb/kb.ts

```diff
@@ -356,6 +536,8 @@ export class WritableKnowledgeBox extends KnowledgeBox implements IWritableKnowl
     return upload(this.nuclia, `/kb/${this.id}`, data, !!TUS, metadata);
   }
 
+  /** Uploads a list of files to the Knowledge Box. It automatically creates a new resource for each file
```

Comment:
Removing the `s` to be consistent with the rest of the doc.
```suggestion
  /** Upload a list of files to the Knowledge Box. It automatically creates a new resource for each file
```